.github/workflows/test.yaml

```diff
@@ -50,3 +50,23 @@ jobs:
         run: npm run format-check --prefix assets
       - name: Run assets tests
         run: npm test --prefix assets
+  # Verify that the app build builds as expected
+  app:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - run: brew install elixir
+      - run: mix local.hex --force
+      - name: Cache Mix
+        uses: actions/cache@v2
+        with:
+          path: |
+            deps
+            _build
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-mix-
+      - name: Install mix dependencies
+        run: mix deps.get
+      - name: Build the app
+        run: .github/scripts/app/build_mac.sh
```

Comment:
running this script is gonna be slooooow since it builds wx, otp, and downloads elixir. Should we cache them? Currently these things are stored into `tmp/` which is also used for some other temporary things so I don't think we should cache `tmp` but maybe move some things around to cache the subset of thereof?

Reply:
Exactly that, I want to see if it works and how long it takes for the record. It works, we should definitely cache those and a more specific directory sounds good!